contracts/deposit-contract/ETH1logs.go

```diff
@@ -23,28 +23,9 @@ func UnpackDepositLogData(data []byte) (pubkey []byte, withdrawalCredentials []b
 		&signature,
 		&index,
 	}
-	if err := contractAbi.Unpack(&unpackedLogs, "Deposit", data); err != nil {
+	if err := contractAbi.Unpack(&unpackedLogs, "DepositEvent", data); err != nil {
 		return nil, nil, nil, nil, nil, fmt.Errorf("unable to unpack logs: %v", err)
 	}
 
 	return pubkey, withdrawalCredentials, amount, signature, index, nil
```

Comment:
is it possible to get time here?

Reply:
No its not possible, the contract doesn't log the timestamp. But you can get the time from the deposit log. It will have the eth1 blockhash, and using that you can get the timestamp for the block/log/deposit

Reply:
thanks